auth/auth_openshift.go

```diff
@@ -88,7 +88,7 @@ func newOpenShiftAuth(ctx context.Context, c *openShiftConfig) (oauth2.Endpoint,
 	}
 
 	// Make sure we can talk to the token endpoint.
-	req, err = http.NewRequest(http.MethodHead, metadata.Token, nil)
+	req, err = http.NewRequest(http.MethodHead, metadata.Token + "/implicit", nil)
```

Comment:
Do we need to check that the token endpoint doesn't end in a slash?

Reply:
It cannot.